src/Doctors/Domain/DataTransferObjects/DoctorDto.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Doctors\Domain\DataTransferObjects;
+
+class DoctorDto
+{
+    public function __construct(
+        public readonly string $name,
```

Comment:
If all attributes are gonna be readonly and you're not extending from a parent class that's not readonly, you should make the class readonly to avoid repeating it for every attribute

```suggestion
readonly class DoctorDto
{
    public function __construct(
        public string $name,
```